Ollivanders/src/net/pottercraft/ollivanders2/Ollivanders2Common.java

```diff
@@ -1106,4 +1119,17 @@ public void sendTitleMessage(String title, String subtitle, ArrayList<Player> pl
 
       return sortedPlayers;
    }
+
+   /**
+    * Determine if a material is a sign. With 1.14 this became hard because there are signs per wood type.
+    *
+    * @param material the material to check
+    * @return true if it is a sign, false otherwise
+    */
+   public boolean materialIsASign(Material material)
+   {
+      return material == Material.ACACIA_WALL_SIGN || material == Material.ACACIA_SIGN || material == Material.BIRCH_SIGN || material == Material.BIRCH_WALL_SIGN
```

Comment:
Can't we do [acacia wall sign, acacia sign, birch wall sign, birch sign ...].contains(material)?

Reply:
changed to use a static arraylist